.github/workflows/build_test.yml

```diff
@@ -180,7 +180,7 @@ jobs:
            download_name: linux
          - image_name: macos-latest
            download_name: macos
-         - image_name: windows-latest
+         - image_name: windows-2019
```

Comment:
`windows-latest` was recently updated to `windows-2022`
This seems to be causing `test_connection.test_invalid_proxy` to hang on the new Windows machines, I believe that these machines let `localhost:3333` hang instead of refusing connection.
I've filed SNOW-541832 to follow up on this.